api/application.ts

```diff
@@ -20,26 +25,23 @@ import routes from './routes'
 import {router as termsAndCRoutes} from './termsAndConditions/routes'
 import {router as userTandCRoutes} from './userTermsAndConditions/routes'
 
-config.environment = process.env.XUI_ENV || 'local'
-
 export const app = express()
 app.disable('x-powered-by')
 
 const FileStore = sessionFileStore(session)
 
 app.set('trust proxy', 1)
-
 app.use(
     session({
         cookie: {
             httpOnly: true,
             maxAge: 1800000,
-            secure: config.secureCookie !== false,
+            secure: getConfigValue(SECURE_COOKIE) !== false,
```

Comment:
having the
```javascript
!== false
```
doesn't make much sense anymore, you can remove it if you wish (the values are now json objects, so true booleans or at least they should be)

Reply:
Done